bin/inx-chronicle/src/metrics.rs

```diff
@@ -89,9 +89,13 @@ impl Actor for MetricsWorker {
 
         let process_metrics_handle = {
             tokio::spawn(async move {
+                if let Err(e) = metrics.update().await {
```

Comment:
Shouldn't the task just exit in this case?

Reply:
Not sure how likely is to have a one time error that never repeats like `proc` files being unavailable. To be honest I never expected this to fail at all.

Reply:
I'm confused. If this fails the first time, won't it just fail over and over in the loop?

Reply:
I am not sure. I'd believe that it is possible that `update` could fail the first time and not fail later but I don't have any scenario to prove it either.